src/Compilers/CSharp/Test/Symbol/Symbols/CovariantReturnTests.cs

```diff
@@ -192,16 +263,34 @@ void M(Base b, Derived d)
                 Diagnostic(ErrorCode.ERR_FeatureInPreview, "M").WithArguments("covariant returns").WithLocation(8, 28)
                 );
             verify(comp);
+            VerifyAssignments(comp, 2);
             comp = CreateCompilationWithCovariantReturns(source).VerifyDiagnostics(
                 );
             verify(comp);
+            VerifyAssignments(comp, 2);
             verify(CompilationReferenceView(comp));
             verify(MetadataView(comp));
```

Comment:
>verify(MetadataView(comp)); [](start = 12, length = 27)

I think we should strengthen the tests and create metadata and retargeted view with ```Program``` class as well (rather than with an empty source) and verify assignments and IL for them too. This applies to other similar tests too.